tests/test_ble_utils_service_targets.py

```
@@ -399,15 +408,22 @@ def test_reconnection_schedule_thread_not_started_and_policy_missing_method() ->
+            create_thread=lambda **_kwargs: SimpleNamespace(
+                ident=None, is_alive=lambda: False
+            ),
             start_thread=lambda _thread: None,
         ),
-        interface=SimpleNamespace(_is_connection_closing=False, _can_initiate_connection=True),
+        interface=SimpleNamespace(
+            _is_connection_closing=False, _can_initiate_connection=True
+        ),
     )
 
     assert scheduler._schedule_reconnect(True, Event()) is False
```

Ruff (0.15.6)

[warning] Boolean positional value in function call

(FBT003)